js/modules/loki_public_chat_api.js

```diff
@@ -574,6 +578,33 @@ class LokiPublicChannelAPI {
           return; // Invalid message
         }
 
+        // Duplicate check
+        const isDuplicate = message => {
+          // The username in this case is the users pubKey
+          const sameUsername = message.username === adnMessage.user.username;
+          const sameText = message.text === adnMessage.text;
+          // Don't filter out messages that are too far apart from each other
+          const timestampsSimilar =
+            Math.abs(message.timestamp - timestamp) <= 10000;
```

Comment:
I would just upgrade the `10000` to its own const variable, e.g. at the top of the file.